apps/backend/src/datasources/postgres/GenericTemplateDataSource.ts

```diff
@@ -44,6 +44,7 @@ export default class PostgresGenericTemplateDataSource
 
     const newQuestionary = await this.questionaryDataSource.clone(
       sourceGenericTemplate.questionaryId,
+      undefined,

```

Comment:
Maybe we can swap these in clone() function or pass them as an object to avoid passing undefined.

Reply:
That is great.